core/metadata_static.h

```diff
@@ -198,7 +198,6 @@ class MDLabelStaticInit
         MDL::addLabelAlias(MDL_CLASSIFICATION_INTRACLASS_DISTANCE, "ClassificationIntraclassDistance");
         MDL::addLabel(MDL_COLOR, LABEL_INT, "color");
         MDL::addLabel(MDL_COMMENT, LABEL_STRING, "comment");
-        MDL::addLabel(MDL_COORD_CONSENSUS_SCORE, LABEL_DOUBLE, "consensusCost");
```

Comment:
this is on purpose?

Reply:
Yes, there is a duplicate `addLabel` for `(MDL_COORD_CONSENSUS_SCORE`.